Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -2378,7 +2378,14 @@ class CloneSerializer : public CloneBase {
         write(file.url().string());
         write(file.type());
         write(file.name());
-        write(static_cast<double>(file.lastModifiedOverride().value_or(-1)));
+        if (m_forStorage == SerializationForStorage::No)
```

Comment:
Can't review because the explanation in the commit log is a bit weak. Maybe @beidson can though.

Still I'd recommend writing proper commit logs that explains what the bug was and how it is now addressed.

Reply:
Updated.